Tests/LibGfx/TestICCProfile.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023, Nico Weber <thakis@chromium.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/MappedFile.h>
+#include <LibGfx/ICC/Profile.h>
+#include <LibGfx/PNGLoader.h>
+#include <LibTest/TestCase.h>
+
+TEST_CASE(png)
+{
+    auto file = MUST(Core::MappedFile::map("/res/cursor-themes/Default/wait.f14t100.png"sv));
```

Comment:
I'd suggest you add a dedicated image for this, instead of relying on a random image which might change or disappear any time :)

Reply:
Good idea. Is there s standard location for files that are test inputs?

Reply:
- LibPDF dumped its test files into the test directory directly: https://github.com/SerenityOS/serenity/tree/master/Tests/LibPDF
- LibGL uses a subdir: https://github.com/SerenityOS/serenity/tree/master/Tests/LibGL/reference-images
- ...so does LibCompress: https://github.com/SerenityOS/serenity/tree/master/Tests/LibCompress/brotli-test-files
- ...but not LibVideo: https://github.com/SerenityOS/serenity/tree/master/Tests/LibVideo

Pick one, I guess :smile:
My personal preference would be a subdir, `Tests/LibGfx/icc` maybe.

Reply:
I will update this PR once #17261 is in :)